tests/rest/admin/test_room.py

```diff
@@ -1413,31 +1419,31 @@ def _order_test(
             "joined_local_members", [room_id_1, room_id_2, room_id_3], reverse=True
         )
 
-        _order_test("version", [room_id_1, room_id_2, room_id_3])
-        _order_test("version", [room_id_1, room_id_2, room_id_3], reverse=True)
+        _order_test("version", all_ids)
+        _order_test("version", all_ids, reverse=True)
 
-        _order_test("creator", [room_id_1, room_id_2, room_id_3])
-        _order_test("creator", [room_id_1, room_id_2, room_id_3], reverse=True)
+        _order_test("creator", all_ids)
+        _order_test("creator", all_ids, reverse=True)
 
-        _order_test("encryption", [room_id_1, room_id_2, room_id_3])
-        _order_test("encryption", [room_id_1, room_id_2, room_id_3], reverse=True)
+        _order_test("encryption", all_ids)
+        _order_test("encryption", all_ids, reverse=True)
 
-        _order_test("federatable", [room_id_1, room_id_2, room_id_3])
-        _order_test("federatable", [room_id_1, room_id_2, room_id_3], reverse=True)
+        _order_test("federatable", all_ids)
+        _order_test("federatable", all_ids, reverse=True)
 
-        _order_test("public", [room_id_1, room_id_2, room_id_3])
+        _order_test("public", all_ids)
         # Different sort order of SQlite and PostreSQL
         # _order_test("public", [room_id_3, room_id_2, room_id_1], reverse=True)
 
-        _order_test("join_rules", [room_id_1, room_id_2, room_id_3])
-        _order_test("join_rules", [room_id_1, room_id_2, room_id_3], reverse=True)
+        _order_test("join_rules", all_ids)
+        _order_test("join_rules", all_ids, reverse=True)
 
-        _order_test("guest_access", [room_id_1, room_id_2, room_id_3])
-        _order_test("guest_access", [room_id_1, room_id_2, room_id_3], reverse=True)
+        _order_test("guest_access", all_ids)
+        _order_test("guest_access", all_ids, reverse=True)
```

Comment:
Are we okay with the order not reversing if we fall back to the tie-breaker and ask for items to be listed in reverse order? It's not what I'd expect, but I don't know if it makes a huge difference when writing an admin frontend.